core/src/epicli/cli/engine/PatchEngine.py

```diff
@@ -20,27 +43,116 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, traceback):
         super().__exit__(exc_type, exc_value, traceback)
 
+    def _validate_input_doc(self, document):
```

Comment:
We cant run this normally through the SchemaValidator?

Reply:
I tried, but it wasn't working for me, it was trying to validate "everything". I'll look again into it and see what I can do. :+1: